File: condicoes/1041.py
#1041 COORDENADAS DE UM PONTO
valores = input().split(" ")
valorX = float(valores[0])
valorY = float(valores[1])

if valorX == 0.0 and valorY == 0.0:
    print("Origem")
elif valorX == 0.0:
    print("Eixo Y")
elif valorY == 0.0:
    print("Eixo X")
elif valorX > 0.0 and valorY > 0.0:
    print("Q1")
elif valorX < 0.0 and valorY > 0.0:
    print("Q2")
elif valorX < 0.0 and valorY < 0.0:
    print("Q3")
elif valorX > 0.0 and valorY < 0.0:
    print("Q4")

File: javascript/1041.js
/*Coordenadas de um Pontos*/
let input = require("fs").readFileSync("/dev/stdin", "utf8");

let lines = input.split(" ");

let X = Number(lines.shift());
let Y = Number(lines.shift());

if(X === 0.0 && Y === 0.0){
  console.log("Origem");
}else if(X == 0.0){
  console.log("Eixo Y");
}else if(Y == 0.0){
  console.log("Eixo X");
}else if(X > 0.0 && Y > 0.0){
  console.log("Q1");
}else if(X < 0.0 && Y > 0.0){
  console.log("Q2");
}else if(X < 0.0 && Y < 0.0){
  console.log("Q3");
}else if(X > 0.0 && Y < 0.0){
  console.log("Q4");
}